[DOCS] Organize breaking changes by component (#79162)
elastic/elasticsearch: docs/reference/migration/migrate_8_0.asciidoc

{ref}/transient-settings-migration-guide.html[Transient settings migration
guide]. 
====
//end::notable-breaking-changes[]

[discrete]
[[breaking_80_command_line_tool_deprecations]]
==== Command line tool deprecations

TIP: {ess-skip-section}

[[deprecate-elasticsearch-setup-passwords]]
.The `elasticsearch-setup-passwords` tool is deprecated.
[%collapsible]
====
*Details* +

> In 8.0, we're deprecating the `elasticsearch-setup-passwords` tool. To
> manually reset the password for built-in users (including the `elastic` user), use
> the {ref}/reset-password.html[`elasticsearch-reset-password`] tool, the {es}
> {ref}/security-api-change-password.html[change passwords API], or the
> User Management features in {kib}. We will remove the
> `elasticsearch-setup-passwords` tool in a future release.

Reviewer: ```suggestion
The `elasticsearch-setup-passwords` tool is deprecated in 8.0. To
manually reset the password for built-in users (including the `elastic` user), use
the {ref}/reset-password.html[`elasticsearch-reset-password`] tool, the {es}
{ref}/security-api-change-password.html[change passwords API], or the
User Management features in {kib}. 
`elasticsearch-setup-passwords` will be removed in a future release.
```

*Impact* +

> When starting {es} for the first time, passwords are generated automatically for
> the `elastic` user. If you run the `elasticsearch-setup-passwords` tool after
> starting {es}, the command will fail  because the password for the `elastic`
> user is already configured.

Reviewer: ```suggestion
Passwords are generated automatically for the `elastic` user when you start {es} for the first time. If you run `elasticsearch-setup-passwords` after
starting {es}, it will fail because the `elastic`
user password is already configured.
```
